helm_deploy/templates/deployment-worker.yaml

```diff
@@ -151,7 +138,7 @@ spec:
             - name: RAILS_LOG_TO_STDOUT
               value: enabled
             - name: ENV
-              value: {{ .Values.secrets.environment}}
+              value: {{ .Values.variables.environment}}
```

Comment:
**nit** variables feels like it could become a catch all config section for our future selves. 

`environment` could be top level and appStoreUrl could be `appStore.Url`

Reply:
also, i know this already existed, but just to flag that in rails `ENV['RAILS_ENV']` is always `production` on a hosted "environment". The term `host` or `host_env` is more informative to distinguish hosted environments and rails environments IMO.

Reply:
yeah I feel like we cross that bridge when we get there in terms of having more environment variables but I agree with your point